src/main/cpp/t06_homework.cpp

```diff
@@ -80,9 +80,76 @@
 
 #include "t06_homework.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
-int t06_homework() {
+bool samewrd(string wrd1, string wrd2, int lgt){
+    bool ex = false;
+    bool wrn = false;
+    int a=0, b=0;
+    for (int i=0; i<lgt; i++){
+        a = wrd1[i];
+        b = wrd2[i];
+        if (abs(a - b) != 0 and abs(a - b) != 32){
+            wrn = true;
+        }
+    }
+    if (!wrn) {
+        ex = true;
+    }
+    return ex;
+}
 
+int t06_homework() {
+    int n=0;
+    cin >>n;
+    set<string> voc; //Сет словаря
+    string st; //Строка для ввода
+    for (int i=0; i<n; i++){
+        cin >>st;
+        voc.insert(st);
+    }
+    multiset<string> wrk; //Мультисет работы
+    while (cin>>st){
+        wrk.insert(st);
+    }
+    /*cin >>n;
+    for (int i=0; i<n; i++){
+        cin >>st;
+        wrk.insert(st);
+    }*/
+    int wrn=0; //Кол-во ошибок
+    int lgt=0; //Длинна слова
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'lgt' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4593725)